rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
```

Comment:
Is there a name for this in system settings? Object scaling percentage?

Reply:
It is called 'custom scaling' in the OS. I found 'font choices' on our style page and chose Monospaced font for the Windows system setting names now. I hope this is ok... (See Slack for Screenshots.)